08_files_lab/03.py

```diff
@@ -1,3 +1,9 @@
-# Given a CSV file with numbers in the second column - 
-# print the sum of the numbers.
-#
+""" add to numbers from csv file"""
+import os,sys
+import csv
+sum = 0
+with open ('c:\python/week_2_2/bool.csv','r') as csvin :

```

Comment:
This works, BUT:

1. Better to prefix path strings with `r`
2. Always open CSV files with `rb`, as that lets the CSV module handle line breaks. Remember some CSV files may have embedded newlines inside cells.